### ISSUE TITLE
/f sell

### ISSUE THREAD

**Issue by Idan330I** [redstone/LegacyFactions#138]
I m  never gonna stop =/
/f sell it would be a new command that sells all the items in the chests around you and the distance would be settable in the config.
Your dear @Idan330I

**Comment by MarkehMe** [comment]
I'm not sure something like this should be added to the `/f` command?

**Comment by Idan330I** [comment]
Yes like the chest in your claimed chunk and it would like remove 1 power from the player that uses it something like that :)
